From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 27 May 2016 10:49:55 +0200
Subject: [PATCH] missing: add SMACK_MAGIC, EFIVARFS_MAGIC, BINFMTFS_MAGIC

These were all introduced before kernel v3.9

Not for upstream, but needed for host-systemd.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/basic/missing.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/src/basic/missing.h b/src/basic/missing.h
index fbe2e26e0132..b875be52d299 100644
--- a/src/basic/missing.h
+++ b/src/basic/missing.h
@@ -959,6 +959,16 @@ static inline int syncfs(int fd)
 }
 #endif
 
+#ifndef SMACK_MAGIC
+#define SMACK_MAGIC 0x43415d53
+#endif
+#ifndef EFIVARFS_MAGIC
+#define EFIVARFS_MAGIC 0xde5e81e4
+#endif
+#ifndef BINFMTFS_MAGIC
+#define BINFMTFS_MAGIC 0x42494e4d
+#endif
+
 #ifndef IPV6_UNICAST_IF
 #define IPV6_UNICAST_IF 76
 #endif
